app/controllers/conference_registrations_controller.rb

```diff
@@ -89,9 +87,9 @@ def destroy
       redirect_to root_path,
                   notice: "You are not registered for #{@conference.title} anymore!"
     else
-      redirect_to root_path,
-                  error: "Could not update your registration for #{@conference.title}: "\
-                  "#{@registration.errors.full_messages.join('. ')}."
+      redirect_to conference_conference_registrations_path(@conference.short_title),
```

Comment:
Why do you redirect to conference_conference_registrations_path? conference_conference_registration_path would be better from my POV...